pkgs/sops-install-secrets/main.go

```diff
@@ -102,6 +103,30 @@ type appContext struct {
 	checkMode   CheckMode
 }
 
+func secureSymlinkChown(symlinkToCheck, expectedTarget string, owner, group int) error {
+	fd, err := unix.Open(symlinkToCheck, unix.O_CLOEXEC|unix.O_PATH|unix.O_NOFOLLOW, 0)
+	if err != nil {
+			return fmt.Errorf("Failed to open %s: %w", symlinkToCheck, err)
+	}
+	defer unix.Close(fd)
+
+	buf := make([]byte, len(expectedTarget) + 1)
+	n, err := unix.Readlinkat(fd, "", buf)
+	if err != nil {
+		return fmt.Errorf("couldn't readlinkat %s", symlinkToCheck)
+	}
+	actualTarget := string(buf[:n]) // chop off null byte
+
+	if !strings.EqualFold(expectedTarget, actualTarget) {
```

Comment:
Linux filesystems should not perform unicode normalisation. So why not compare bytes here?

Reply:
Mostly a leftover from troubleshooting the aforementioned issue, I'll switch this back to a regular `==`.